README.md

```diff
@@ -250,6 +269,43 @@ the query result like below:
 }
 ```
 
+### Pub message and retrieve ZKP by `ioctl`
+
+```sh
+ioctl ws message send --project-id ${project id} --project-version ${project version} --data ${message content}
+ioctl ws message query --message-id ${message id}
+```
+
+### w3bstream project management
+
+You need deploy project register contract to **IOTX** before operating w3bstream projects.
```

Comment:
maybe we can give users contract address we have deployed, and users  just write the contract and don't need deploy the contract themself

Reply:
the `project register contract address` is a command parameter supporting testing.